obs-browser-page/obs-browser-page-main.cpp

```diff
@@ -34,13 +34,22 @@ static HANDLE shutdown_event = nullptr;
 static bool thread_initialized = false;
 
 DECLARE_HANDLE(OBS_DPI_AWARENESS_CONTEXT);
+#if defined(_M_ARM64)
+	#define OBS_DPI_AWARENESS_CONTEXT_UNAWARE ((OBS_DPI_AWARENESS_CONTEXT) - 1)
+	#define OBS_DPI_AWARENESS_CONTEXT_SYSTEM_AWARE ((OBS_DPI_AWARENESS_CONTEXT) - 2)
+	#define OBS_DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE \
+		((OBS_DPI_AWARENESS_CONTEXT) - 3)
+	#define OBS_DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2 \
+		((OBS_DPI_AWARENESS_CONTEXT) - 4)
+#else
```

Comment:
Please elaborate on this change. The only difference seems to be indentation and spacing?

Reply:
Hi @WizardCM 
Thank you for pointing that out. It was a mistake, I have modified it now